server/daemon/src/main.rs

```diff
@@ -785,18 +792,33 @@ async fn kanidm_main(
                                         // Reload TLS certificates
                                         // systemd has a special reload handler for this.
                                         #[cfg(target_os = "linux")]
-                                        let _ = sd_notify::notify(true, &[sd_notify::NotifyState::Reloading]);
+                                        // https://stackoverflow.com/a/70337205
+                                        // is this close enough to CLOCK_MONOTONIC in clock_gettime(2) ?
```

Comment:
For this use case it should be fine, we're not worried about filestomping and things like that.

Reply:
Thanks for confirming!